src/Games/NexusMods.Games.FOMOD/FomodXmlInstaller.cs

```diff
@@ -48,7 +48,13 @@ public Priority GetPriority(GameInstallation installation, EntityDictionary<Rela
         // we only intend to support xml scripted fomods, this should be good enough
         var stopPattern = new List<string> { "fomod" };
 
-        if (!archiveFiles.TryGetValue(FomodConstants.XmlConfigRelativePath, out var analyzedFile))
+        var xmlFileList = archiveFiles.Keys.Where(x => x.EndsWith(FomodConstants.XmlConfigRelativePath));
```

Comment:
This `IEnumerable` will be enumerated multiple times. Use `.ToArray` instead to prevent multiple enumerations.

Reply:
I'll change it use TryGetFirst instead of Where so that I get what I'm looking for directly.